2022q1/doceng.adoc

```diff
@@ -0,0 +1,100 @@
+////
+Quarter:	1st quarter of 2022
+Prepared by:	dbaio
+Reviewed by:	carlavilla
+Last edit:	$Date: 2022-04-09 09:51:36 -0300 (Sat, 09 Apr 2022) $
+Version:	$Id: doceng-2022-1st-quarter-status-report.adoc 221 2022-04-09 12:51:36Z dbaio $
+////
+
+=== Documentation Engineering Team
+
+Link: link:https://www.freebsd.org/docproj/[FreeBSD Documentation Project] +
+Link: link:https://docs.freebsd.org/en/books/fdp-primer/[FreeBSD Documentation Project Primer for New Contributors] +
+Link: link:https://www.freebsd.org/administration/#t-doceng[Documentation Engineering Team]
+
+Contact: FreeBSD Doceng Team <doceng@FreeBSD.org>
+
+The doceng@ team is a body to handle some of the meta-project issues associated with the FreeBSD Documentation Project; for more information, see link:https://www.freebsd.org/internal/doceng/[FreeBSD Doceng Team Charter].
+
+No new documentation commit bit was granted during the last quarter, and only one commit bit was safe kept.
+
+Several tasks were completed related to the doc tree during the last quarter:
+
+* Fix some issues in the translation workflow with PO files and Weblate related to the po4a program.
++
+More info link:https://lists.freebsd.org/archives/freebsd-translators/2022-February/000029.html[here].
+
+* Update link:https://download.freebsd.org/doc/[offline documentation (PDF and HTML)].
++
+The old directory /doc is now on link:http://ftp-archive.freebsd.org/mirror/FreeBSD-Archive/old-docs/[ftp-archive]; it contains files prior Hugo/Asciidoctor migration.
+
+* Remove Google Analytics from documentation and website.
+
+* Add last modified information to the documentation and website pages.
+
+* Tag FreeBSD docset for 13.1-RELEASE.
+
+* Add the first Indonesian translation to the doc tree.
+
+
+==== FreeBSD Translations on Weblate
+
+Link: link:https://wiki.freebsd.org/Doc/Translation/Weblate[Translate FreeBSD on Weblate] +
+Link: link:https://translate-dev.freebsd.org/[FreeBSD Weblate Instance]
+
+The translation workflow with Weblate is more mature at this point.
+Several issues were fixed between PO files and po4a program.
+
+We welcome everyone to try our Weblate instance to translate a few documents.
+
+The first Indonesian translation was added to the FreeBSD project.
+We thank Azrael JD for the contribution, and we are looking forward to seeing more Indonesian translations.
+
+===== Q1 2022 Status
+
+* 12 languages (1 new language)
+* 142 registered users
```

Comment:
Please explain in more detail. This is the first time since I started copyediting status reports this stat is present.

Reply:
https://www.freebsd.org/status/report-2020-10-2020-12.html#FreeBSD-Translations-on-Weblate

This is just a table that will show the difference between each quarter; we stopped including this after the migration to Hugo/Asciidoctor because the workflow wasn't well defined.